### ISSUE TITLE
signal/test_callbackq_async.test failed on bgate

### ISSUE THREAD

**Issue by alex-mikheev** [openucx/ucx#852]
On bgate:
http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2039/

```
16:16:37 [----------] 1 test from signal/test_callbackq_async
16:16:37 [ RUN      ] signal/test_callbackq_async.test/0
16:16:37 [1467033397.855680] [jenkins01:28070:5]      callbackq.c:138  UCX  WARN  1 callbacks still remain in callback queue
16:16:37 [1467033397.856576] [jenkins01:28070:5]      callbackq.c:143  UCX  WARN  cbq 0xc0fd38: remain 0xb500e0 ucs_callbackq_service_cb(arg=0xc0fd38)
16:16:37 /var/lib/jenkins/jobs/gh-ucx-pr/workspace/contrib/../test/gtest/common/test.cc:133: Failure
16:16:37 Failed
16:16:37 Got 2 warnings during the test
16:16:37 [  FAILED  ] signal/test_callbackq_async.test/0, where GetParam() = 0 (301 ms)
16:16:37 [----------] 1 test from signal/test_callbackq_async (301 ms total)
16:16:37 
```

**Comment by yosefe** [comment]
@shamisp did you also use to get a similar failure?

**Comment by shamisp** [comment]
I'm not sure actually

**Comment by shamisp** [comment]
opps. see it now 
efault number of updates (RECOMMENDED) = 1048576
[1467146485.962561] [x:6822 :0]    ib_mlx5_log.c:110  UCX  ERROR Error on QP 0xab6 wqe[2072]: Local QP operation (synd 0x2 vend 0x61) opcode FETCH_ADD
[1467146485.963051] [x:6822 :1]      ib_device.c:132  UCX  WARN  IB Async event on mlx5_0: SRQ-attached QP 0xab6 was flushed

**Comment by yosefe** [comment]
fixed in #1059
